### ISSUE TITLE
Get rid of default_images and default_styles

### ISSUE THREAD

**Issue by pricem** [learning-unlimited/ESP-Website#880]
Now that we are getting rid of file-sharing, we may be able to go back to using the images and styles directories from the repository.  There may be some issues to think about here (beyond just renaming the dirs), particularly when it comes to chapter specific customizations.  We would also need a procedure for converting an existing site over (how do you reconcile any differences between default_images and images)?

**Comment by milescalabresi** [comment]
I think there are two and a half tasks remaining on this issue:

- [ ] 1. Update the code (including our custom scripts and development resources) so that it only ever references `images` and `styles` instead of `default_*`.
- [ ] 2. Delete the symlinked folders in `public/media/`.

For the spirit of the challenge of admins uploading custom images and styles, I think we're addressing that elsewhere. Theme-related images (headers/logos/landing pages) have been solved reasonably well, and I think we are (correctly) moving away from custom stylesheets in favor of theme customizations. Other images can/should be in the `uploaded` folder, which git ignores.

**Comment by Supriyo760** [comment]
Hi @willgearty, I'd like to work on this!
Based on @milescalabresi's 2023 summary, I understand there are two remaining tasks:
1. Update the codebase to remove all references to default_images and default_styles, replacing them with just images and styles
2. Remove the symlinked folders in public/media/

For task 1 - I'd grep the codebase for all default_images and default_styles references across Python files, templates, scripts, and config files, and update them systematically. I'd also verify nothing in the static file pipeline depends on the default_* naming.
For task 2 - could you confirm whether deleting the symlinks requires server-side access (like #3785), or can this be handled via a fabric/management command in the repo? Happy to handle whichever parts are within contributor scope.

If this approach sounds good, could I be assigned to this issue?

**Comment by willgearty** [comment]
That'd be great, thanks! I'm not sure about deleting the symlinks in production. We can probably just add it as a task to our Makefile for when we do the next release (which requires server-side access). @milescalabresi may have thoughts.

**Comment by milescalabresi** [comment]
Agreed. We should also check on the server whether `default_*` is set in any config files that show up in the code as `esp.settings.*`

**Comment by Supriyo760** [comment]
@willgearty  @milescalabresi  
Since deleting the symlinks requires server-side access, my PR will focus entirely on the codebase cleanup aspect. 
Specifically, I will:
Update all codebase references (Python files, HTML templates, scripts) from default_images and default_styles to images and styles.
Ensure there are no lingering default_* configs set anywhere under esp.settings.* as @milescalabresi suggested.
Review the Makefile and try to add a cleanup task there for the production symlinks (if possible).

I'll get a PR up for this shortly!

**Comment by Supriyo760** [comment]
Hi @willgearty and @milescalabresi , I've just opened PR #4458 to resolve Issue #880. Whenever you have a moment, I'd appreciate it if you could take a look. Thank you!

**Comment by milescalabresi** [comment]
Hi @Supriyo760, we will look when we can. Please note we get notified when the PR is created, so there is no need to add a comment to the issue.
